### PR TITLE
(docs) document ApplyResult

### PR DESCRIPTION
**Pull request by adreyer**

### CODE REVIEW

File: pre-docs/applying_manifest_blocks.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+Within a plan, you can use Bolt to apply blocks of Puppet code (manifest blocks) to remote nodes. Similar to the `puppet apply` command, which applies a standalone Puppet manifest to a local system, the Bolt `apply` command leverages manifest blocks to pass code to remote nodes from the command line. You can create manifest blocks that use existing content from the Forge, or mix declarative resource configuration via manifest blocks with procedural orchestration and action in a plan. Most features of the Puppet language are available in a manifest block: classes, custom resource types, and functions. Exceptions are noted in [caveats](#caveats).
```
Comment (lucywyman) [inline pre-docs/applying_manifest_blocks.md]: s/Bolt `apply` command/Bolt `apply` statement/ -- to avoid confusion with command line command

Comment (adreyer) [reply pre-docs/applying_manifest_blocks.md]: I'd like to bikeshed a bit on what we call the apply statement in docs. Command definitely seems wrong.

function?
statement?
maneuver?

Comment (adreyer) [reply pre-docs/applying_manifest_blocks.md]: action

**Comment by adreyer** [comment]
Its not 100% clear how to work "plan action" into this. I think that should be a larger standardization PR.

**Comment by MikaelSmith** [comment]
Should we just merge this then?

**Pull request merged**